### DISCUSSION TITLE
Packaging binaries/applications as ports

### DISCUSSION THREAD

**Discussion by TheTollingBell** [microsoft/vcpkg#39362] [Q&A] [answered]
I wanted to package Doxygen for use in my own project however I realized there is no real way to "provide" the executable to the end user. Is this intentional? Should I not be packaging tools using vcpkg and instead use something else? I came to vcpkg because conan is extremely bloated and hard to use but not being able to use tools would almost be enough to make me switch back.

**Comment by Neumann-A** [comment, marked as answer]
The main registry does not have rules for tools yet so you cannot have it here currently. However, you can have tool ports yourself and if you copy the tool into `<vcpkg_installed>/tools/something` `find_program` will be able to find it.

**Reply by TheTollingBell** [reply]
Thanks!
